src/main/webapp/portal/main/partials/access-denied.html

```diff
@@ -0,0 +1,13 @@
+<frame-page app-header-title='Access denied' 
```

Comment:
That would be nice, might be challenging to do. I guess we could pass a variable via the route variables.